imports/plugins/core/accounts/client/components/CreateOrEditGroupDialog.js

```diff
@@ -0,0 +1,222 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import i18next from "i18next";
+import SimpleSchema from "simpl-schema";
+import CloseIcon from "mdi-material-ui/Close";
+import Button from "@reactioncommerce/catalyst/Button";
+import TextField from "@reactioncommerce/catalyst/TextField";
+import Select from "@reactioncommerce/catalyst/Select";
+import useReactoForm from "reacto-form/cjs/useReactoForm";
+import muiOptions from "reacto-form/cjs/muiOptions";
+import { useMutation } from "@apollo/react-hooks";
+import { useSnackbar } from "notistack";
+import {
+  Box,
+  Grid,
+  CardActions,
+  CardHeader,
+  CardContent,
+  Dialog,
+  IconButton,
+  makeStyles
+} from "@material-ui/core";
+import useRoles from "../hooks/useRoles";
+import createGroupMutation from "../graphql/mutations/createGroup";
+import updateGroupMutation from "../graphql/mutations/updateGroup";
+
+const useStyles = makeStyles((theme) => ({
+  cardRoot: {
```

Comment:
This class needs to be added to the `Card` component that wraps the card content, which is also missing below. Otherwise, the Select options will be cut off, which is the case currently.

Reply:
I just tried this, but it unfortunately doesn't solve the issue of the select options being cut off. Weirdly, `TagSelectorDialog` uses `classes={{ root: classes.cardRoot }}` as a prop to `<Dialog />`, as I did initially, and the select options on there are fine.

Maybe we should be looking somewhere else? I have to admit that I'm not a CSS expert by any means, so I may be missing something obvious.

Reply:
Added suggestions on how to fix.